zerver/templatetags/minified_js.py

```diff
@@ -19,7 +19,13 @@ def __init__(self, sourcefile):
     def render(self, context):
         # type: (Dict[str, Any]) -> str
         if settings.DEBUG:
-            scripts = settings.JS_SPECS[self.sourcefile]['source_filenames']
+            source_files = settings.JS_SPECS[self.sourcefile]
+            normal_source = source_files['source_filenames']
+            minified_source = source_files.get('minifed_source_filenames')
```

Comment:
I would probably use `.get('minified_source_filenames', [])` (so the default if it isn't there is an empty list, not None) and then remove the `if else` a few lines down; it'll be more readable that way.